packages/site/src/locales/en.json

```diff
@@ -143,7 +143,9 @@
     "link_copied": "Link copied",
     "copy_and_share": "Copy and share",
     "sr": "Source",
-    "example_sentence": "Example Sentence"
+    "example_sentence": "Example Sentence",
+    "nc": "Noun Class",
+    "va": "Variant"
```

Comment:
@Danble please spell out translation keys "noun_class" and "variant" unless trying to specifically map to a database key. Are you wanting to update our entry interface - if so let's have a conversation to discuss, otherwise don't abbreviate these keys.

Reply:
@jacob-8 Yes! Anna told me this should be added. Right now it will only be needed in the Babanki dictionary but she thinks more dictionaries will need these fields.